clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -635,7 +635,7 @@ public void testApplicationBufferResize() throws Exception {
     @Test
     public void testNetworkThreadTimeRecorded() throws Exception {
         selector.close();
-        this.selector = new Selector(NetworkReceive.UNLIMITED, 5000, new Metrics(), Time.SYSTEM,
+        this.selector = new Selector(NetworkReceive.UNLIMITED, 10000, new Metrics(), Time.SYSTEM,
```

Comment:
Would it make sense to disable idle timeout with `NO_IDLE_TIMEOUT_MS` for this test? Do we want it to be at play?

Reply:
Agree, We can set timeout to NO_IDLE_TIMEOUT_MS  for this test.